### DISCUSSION TITLE
Update Dutch Language to 0.53.0

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#613] [General]
Hi Rejetto,

Attached the update of the language file to version 0.53.0

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl (if the main computer here is online)

[hfs-lang-nl 0-53-0.zip](https://github.com/user-attachments/files/15747742/hfs-lang-nl.0-53-0.zip)

**Comment by rejetto** [comment]
thanks, included in next version
